### PR TITLE
added coc command 

### PR DESCRIPTION
**Pull request by ajax711**

### CODE REVIEW

File: bot.ini
Comment (hellozee) [inline bot.ini]:
```suggestion
github=https://github.com/ILUGD
```

**Review by rajudev — COMMENTED** [review]
Line number 8 in bot.ini

**Review by hellozee — CHANGES_REQUESTED** [review]
Apart from that single change, the patch looks fine, though can't merge it currently since the COC is still a WIP, has many flaws, as soon as the COC gets finalized, this PR can be merged, :smile_cat:

**Review by hellozee — APPROVED** [review]

**Pull request merged**
